Clarify why partial `${{ }}` wrapping is incorrect in actions-067 (#593)
FidelusAleksander/ghcertified: questions/en/actions/question-067.md

my-job:
  if: ${{ vars.MY_VAR }} == 'MY_VALUE'
```

> > Incorrect, only `vars.MY_VAR` is evaluated inside `${{ }}`; the `== 'MY_VALUE'` part is left as literal text, so the result can be a non-empty string, which is truthy in an `if`

Reviewer: ```suggestion
> The `== 'MY_VALUE'` part is left as literal text, so the result can be a non-empty string, which is truthy in an `if` conditional
> Read more here https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#literals
```

> > Incorrect, only `vars.MY_VAR` is evaluated inside `${{ }}`; this yields text like `some_value == 'MY_VALUE'`, and GitHub treats that non-empty string as truthy instead of performing the comparison

Reviewer: ```suggestion
> Incorrect, only `vars.MY_VAR` is evaluated inside `${{ }}`; this yields text like `some_value == 'MY_VALUE'`, and GitHub treats that non-empty string as truthy instead of performing the comparison
> See https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#literals
```
